src/lib/DeployUtil.ts

```diff
@@ -137,20 +63,22 @@ export class UniqAddress {
   transferId: BigNumber;
 
   /**
-   * Constructs UniqAddress from the transacting account's `CLPublicKey` and unique transferId.
+   * Builds UniqAddress from the transacting account's `CLPublicKey` and unique transferId.
    * @param publicKey CLPublicKey instance
    * @param transferId BigNumberish value (can be also string representing number). Max U64.
    */
-  constructor(publicKey: CLPublicKey, transferId: BigNumberish) {
```

Comment:
Types used with @jsonObject and TypedJson generally require a no-args constructor